<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_durability">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>durability</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_weight">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>weight</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_attack">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>attack</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_defense">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>defense</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_durability">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_weight">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_attack">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_defense">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_durability_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_weight_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_attack_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_defense_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Минимальная</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Максимальная</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Характеристика</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
